packages/assets/src/scss/mixins/_selection-inputs.scss

```
@@ -0,0 +1,13 @@
+@use '../functions' as *;
+@use '../variables' as *;
+
+@mixin selection-input-base {
```

This name is a bit confusing to me. My first thought was "select," not "radio" or "checkbox."
```suggestion
@mixin checkable-input-base {
```
WDYT?